src/ui/common/src/components/workflows/workflowHeader.tsx

```diff
@@ -69,7 +69,8 @@ const WorkflowHeader: React.FC<Props> = ({ user, workflowDag, workflowId }) => {
     }
   };
 
-  useEffect(getContainerSize, [currentNode]);
+  // TODO: useLayoutEffect here. May want to figure out some way to debounce as it gets called quite quickly when resizing.
```

Comment:
Can we add a linear task in `TODO(task code)` format here?

Reply:
Done!

https://linear.app/aqueducthq/issue/ENG-2302/use-eayout-effect-when-handling-window-resizing-in-workflow-header